R/r_util.R

```diff
@@ -1,6 +1,21 @@
-CLIGetParamMat <- function(paramName, paramValue) {
-  mat <- CLI_GetParamMat(as.character(paramName))
-  nrows <- CLI_GetParamMatRows(as.character(paramName))
-  ncols <- CLI_GetParamMatCols(as.character(paramName))
-  return(matrix(mat, nrows, ncols))
+CLIGetParamMat <- function(param_name, points_are_rows) {
```

Comment:
These two are the last two uses of `points_are_rows`.  Should we add a default value in the signature to get default behavior without having to set it?

Reply:
From @eddelbuettel's later comment, the default value would be: 

```r
CLIGetParamMat <- function(param_name, 
                                                 points_are_rows = getOption('rcppmlpack.tranpose', TRUE)
                                                 )
```

If more are needed, we probably will establish an environment. But, that can be handled at a later time.

Reply:
I am really sorry. I didn't remove this file in the last commit. But this file is of no use now. As we have decided that we always have to transpose the matrix(before/after), Hence I transferred all the translation to RcppArmadillo because it performed better in microbenchmark for the translation.